packages/page-statistics/src/containers/KeygenTable/KeygenTable.tsx

```diff
@@ -140,10 +147,9 @@ export const KeygenTable: FC = () => {
     [pageSize, pagination.pageIndex, pagination.pageSize]
   );
 
-  const pageCount = useMemo(() => Math.floor(totalItems / pageSize), [pageSize, totalItems]);
+  const pageCount = useMemo(() => Math.round(totalItems / pageSize), [pageSize, totalItems]);
```

Comment:
```suggestion
  const pageCount = useMemo(() => Math.ceil(totalItems / pageSize), [pageSize, totalItems]);
```

Reply:
This happened because last time we didn't handle the `null` case so some latest sessions may contain `null` values. But at the moment if we have dealt with the `null` case, we should update the `pageCount` calculation.